aggregate/Pulumi.billing_aggregate.yaml

```diff
@@ -1,12 +1,15 @@
 config:
   opts:name: aggregate-billing
+  opts:type: cloud-run
+  opts:dockerfile: billing_functions/Dockerfile
+  opts:imagename: aggregate-billing-image
   opts:service_account: aggregate-billing@billing-admin-290403.iam.gserviceaccount.com
   opts:functions:
     - gcp
     - hail
     - seqr
-  opts:memory: 1024
-  opts:timeout: 540
+  opts:memory: 8192
```

Comment:
Wow, that's a lot. Do we know why it uses so much memory?

Reply:
I'm not sure what the most memory it uses is. It heavily depends on how much data needs to be loaded each day. My plan is to set it high, see how it runs and then set a more realistic max memory cap.